modules/machine-user-infra-machines-pxe.adoc

```diff
@@ -1,6 +1,7 @@
 // Module included in the following assemblies:
 //
 // * machine_management/user_infra/adding-bare-metal-compute-user-infra.adoc
+// * post_installation_configuration/node-tasks.adoc
```

Comment:
Can you update the approving CSRs module to also include this assembly in the metadata?

Reply:
@bergerhoffer The _post_installation_configuration/node-tasks.adoc_ assembly was already in the _installation-approve-csrs_ module.